include/xsimd/config/xsimd_cpuid.hpp

```diff
@@ -141,6 +141,12 @@ namespace xsimd
                 if (sse4_2)
                     best = std::max(best, fma3<xsimd::sse4_2>::version() * fma3_sse);
 
+                avx = regs[2] >> 28 & 1;
```

Comment:
I don't understand the fix, can you explain?

Reply:
regs is used before being overwritten with the call o `get_cpuid(regs, 0x80000001);`

Reply:
ok. Maybe worth having separate locale variable for each call to `get_cpuid` then, right?

Reply:
Indeed.